blueocean-core-js/src/js/fetch.js

```diff
@@ -290,7 +290,10 @@ function getPrefetchedDataFuture(url) {
                 // the prefetched data as the value.
                 if (trimRestUrl(preFetchEntry.restUrl) === trimmedUrl) {
                     try {
-                        return Promise.resolve(JSON.parse(preFetchEntry.data));
+                        const parse = JSON.parse(preFetchEntry.data);
+                        return Promise.resolve(parse);
+                    } catch (e) {
+                        console.error('preFetch Error', e);
```

Comment:
Could we rethrow a new error in the catch block which has the prefetch.data attached to it? So the details show up in rollbar?  (unless console.errors show up automatically, but I didn't think they do)

```
} catch (e) {
    console.error('preFetch Error', e);
    const err = new Error();
    error.prefetchData = preFetchEntry.data;
    throw err;
}
```